insights/parsers/dnf_module.py

```diff
@@ -17,22 +17,37 @@
 from insights.specs import Specs
 
 
+class DnfModuleStream(object):
+    """
+    An object for the dnf module:stream
+
+    Attributes:
+        name (str): Name of the module stream
+        profiles (list): List of profiles of the dnf module:stream
+        summary (str): Summary of the dnf module:stream
+    """
+    def __init__(self, data=None):
+        data = data or {}
+        self.name = data.get('Stream', '')
```

Comment:
What do you think about parsing the special bracket flags in this parser?

Reply:
Yeah, I wanted to do it but then I decided to only fix the bug instead of refactoring the whole module